api/queries_pr.go

```diff
@@ -185,7 +186,7 @@ func PullRequests(client *Client, repo ghrepo.Interface, currentPRNumber int, cu
 	queryPrefix := `
 	query($owner: String!, $repo: String!, $headRefName: String!, $viewerQuery: String!, $reviewerQuery: String!, $per_page: Int = 10) {
 		repository(owner: $owner, name: $repo) {
-			pullRequests(headRefName: $headRefName, states: OPEN, first: $per_page) {
+			pullRequests(headRefName: $headRefName, first: $per_page) {
```

Comment:
Maybe we should also define an `order` clause here to sort PRs newest first?

Reply:
Define an order clause. https://github.com/cli/cli/pull/530/commits/9c44eaeb6ec93937b8b62fd74e9954557c5f2aa6
Not sure how I can simulate a situation that it matters. 🤔

Reply:
@doi-t I was thinking that the situation where it matters is when there is both an open PR and some closed PRs with the head branch named "foo", and we want to prioritize the latest (open) one.

Reply:
@mislav 
Turned out the suggested query messes up the result of `gh pr status` if there are multiple PRs with the head branch named "foo". Will try to figure out an appropriate query...

#### A result of `orderBy: { field: CREATED_AT, direction: ASC }`
`#6` should be in `Current branch`.
<img width="304" alt="Screenshot 2020-02-25 at 14 36 01" src="https://user-images.githubusercontent.com/5877477/75219562-74144e00-57e0-11ea-8a65-220ef73d7b89.png">

#### A result of `orderBy: { field: CREATED_AT, direction: ASC }`
`#5` is gone and `#6` got duplicated.
<img width="304" alt="Screenshot 2020-02-25 at 14 37 52" src="https://user-images.githubusercontent.com/5877477/75219568-79719880-57e0-11ea-87f8-6610d912063a.png">